doc/doxygen/headers/c++.h

```diff
@@ -82,44 +79,36 @@
  * @code
  * constexpr Tensor<2, 2> A({{1., 0.}, {0., 1.}});
  * @endcode
- * Some functions such as determinant() are specified as `constexpr` but they
- * require a compiler with C++14 capability. As such, this function is
- * internally declared as:
+ * Some functions such as determinant() are specified as `constexpr`: these rely
+ * on the generalized constexpr support available in C++14. Some functions,
+ * such as deviator_tensor(), rely on further developments of `constexpr` only
+ * available in C++17 and newer. As such, this function is declared as
  * @code
  * template <int dim, typename Number>
- * DEAL_II_CONSTEXPR Number determinant(const Tensor<2, dim, Number> &t);
+ * DEAL_II_CONSTEXPR inline SymmetricTensor<2, dim, Number>
+ * unit_symmetric_tensor();
```

Comment:
It's a  little bit strange that you use `unit_symmetric_tensor()` here and `deviator_tensor()` in the text.

Reply:
Yes, that was a mistake. I fixed it.